detect-monero-demo/README.md

```diff
@@ -0,0 +1,57 @@
+# Detect Monero Miners with bpftrace
+This demo accompanies the [Detect Monero Miners with bpftrace blogpost]().
+
+## Deploying xmrig to your cluster
+**Cloud Providers have strict policies around cryptomining. We strongly recommend not deploying
+to a cloud provider or you will risk account deactivation.**
```

Comment:
remove newline

Reply:
newline doesn't render anyways. Gonna keep it for local dev readability.